mei40To50/mei40To50.xsl

```diff
@@ -585,12 +585,34 @@
     <!-- SELF-DOCUMENTATION TEMPLATES                                            -->
     <!-- ======================================================================= -->
     
+    <xd:doc>
+        <xd:desc>
+            <xd:p>Documentation for files without top-level mei:meiHead.</xd:p>
+            <xd:p>Matches root elements that are not mei:meiHead and do not contain an mei:meiHead element as direct child.</xd:p>
+        </xd:desc>
+        <xd:return>Inserts XML-comments for documentation then processes the XML-tree.</xd:return>
+    </xd:doc>
+    <xsl:template match="/mei:*[not(mei:meiHead) and not(self::mei:meiHead)]">
+        <xsl:variable name="documentation-change" as="element()">
+            <xsl:call-template name="revisionDesc-insert-change"/>
+        </xsl:variable>
+        <xsl:variable name="documentation-application" as="element()">
+            <xsl:call-template name="appInfo-insert-current-application"/>
+        </xsl:variable>
+        <xsl:comment>
+            <xsl:value-of select="$documentation-change/mei:date/@isodate, $documentation-change/mei:changeDesc/mei:p" separator=" – "/>
```

Comment:
For the isodate, the 40to50 conversion uses only date (L638 following):

```
<xsl:attribute name="isodate">
    <xsl:value-of select="format-date(current-date(), '[Y]-[M02]-[D02]')"/>
</xsl:attribute>
```

The linter uses dateTime (L267 following):

```
<xsl:element name="date">
        <xsl:attribute name="isodate" select="format-dateTime(current-dateTime(), '[Y]-[M,2]-[D,2]T[H]:[m]:[s][Z]')"/>    
</xsl:element>

```

Maybe we could unify this, too?

Reply:
Thanks for spotting this!
Happy to unify it: which one do you prefer?

Reply:
Not sure. But if I see it correctly, none of the other scripts except for the linter used dateTime?

Reply:
pushed a commit with corresponding changes